doc/whatsnew/2.0.rst

```diff
@@ -25,6 +25,18 @@ Summary -- Release highlights
 
 New checkers
 ============
+* A new check was added, ``useless-object-inheritance``.
+
+  This refactoring message is emitted when pylint detects that a class inherits from object,
+  which is redundant as in Python 3, every class implicitly inherits from object.
```

Comment:
`where every class implicitly`

Reply:
"which is redundant as in Python 3, where every class implicitly inherits from object." seems not correct to me grammatically to me, native speaker can comment on this. :smile: